src/data_loader/mnist_result.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+from torch.utils.data import Dataset
+
+
+class MnistResultDataset(Dataset):
+    """
+    Customized MNIST result dataset demo
+    """
+    def __init__(self, result_filename):
+        self.results = self._load_data(result_filename)
+
+    def _load_data(self, result_filename):
+        return np.load(result_filename)['model_output']
```

Comment:
Key `model_output` should be configurable since the model might change its output key anytime

Reply:
Fixed